stable/ambassador/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.50.1
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 1.1.0
+version: 1.1.1
```

Comment:
Can you add an `icon: https://www.getambassador.io/images/logo.png` field while you're at it?

Reply:
Done!